# === EMCL ===
# flag
flag_init_noise: true

# particle
particle_num: 420

# initial pose
init_x: -2.0
init_y: -0.5
init_yaw: 0.0
init_x_dev: 0.1
init_y_dev: 0.1
init_yaw_dev: 0.05

# resetting
alpha_th: 0.0017
reset_count_limit: 5
expansion_x_dev: 0.08
expansion_y_dev: 0.05
expansion_yaw_dev: 0.01

# sensor
laser_step: 4
sensor_noise_ratio: 0.02


# === OdomModel ===
ff: 0.17
fr: 0.0005
rf: 0.13
rr: 0.2
